ext/fetch/20_headers.js

```diff
@@ -286,6 +286,11 @@ class Headers {
 
   /** @param {HeadersInit} [init] */
   constructor(init = undefined) {
+    if (init === webidl.brand) {
+      this[webidl.brand] = webidl.brand;
```

Comment:
Is `this[init] = init` faster? It should have two less property lookups

Reply:
Good catch, missed that.